src/core/tests/Test_AptitudePackageManager.py

```diff
@@ -172,7 +172,7 @@ def test_install_package_only_upgrades(self):
         self.assertIsNotNone(package_manager)
 
         # test for unsuccessfully installing a package
-        self.assertEqual(package_manager.install_update_and_dependencies('iucode-tool', '1.5.1-1ubuntu0.1', simulate=True), Constants.PENDING)
+        self.assertEqual(package_manager.install_update_and_dependencies_and_get_status('iucode-tool', '1.5.1-1ubuntu0.1', True), Constants.PENDING)
```

Comment:
@rane-rajasi  very nit - simulate=true for readability like other similar calls.